CMakeLists.txt

```diff
@@ -571,9 +579,15 @@ elseif("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "CYGWIN")
       "Deployment OS for Swift host tools (the compiler) [windows].")
 
   configure_sdk_unix(CYGWIN "Cygwin" "windows" "cygwin" "windows" "x86_64-unknown-windows-cygnus" "/")
-  set(SWIFT_PRIMARY_VARIANT_SDK_default  "${SWIFT_HOST_VARIANT_SDK}")
+  set(SWIFT_PRIMARY_VARIANT_SDK_default "${SWIFT_HOST_VARIANT_SDK}")
   set(SWIFT_PRIMARY_VARIANT_ARCH_default "x86_64")
 
+elseif("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "WINDOWS")
+  configure_sdk_windows(WINDOWS "Windows" "pc" "msvc" "${SWIFT_HOST_VARIANT_ARCH}")
```

Comment:
Right. Once we get to the awesome point of compiling the stdlib and the compiler, I ran `swiftc helloworld.swift`, and got an error that goes something like this:
`could not find stdlib x86_64-pc-windows-msvc`

updating `unknown` to `pc` fixes this